tests/Makefile

```diff
@@ -8,4 +8,9 @@ tests-catapult-integration:
 tests-catapult:
 	./tests_catapult.sh
 
-all: tests-catapult tests-catapult-integration
\ No newline at end of file
+.PHONY: lint
+lint:
+	./lint.sh
+
+.PHONY: all
+all: lint tests-catapult tests-catapult-integration
```

Comment:
Leave it off as default until we figure out why fails on travis, so we can merge anyway :)